test/test_helper.exs

```diff
@@ -4,8 +4,7 @@ ExUnit.configure exclude: [integration: true, consumer_group: true, server_0_p_9
 
 defmodule TestHelper do
   def generate_random_string(string_length \\ 20) do
-    :random.seed(:os.timestamp)
-    Enum.map(1..string_length, fn _ -> (:random.uniform * 25 + 65) |> round end) |> to_string
+    Enum.map(1..string_length, fn _ -> (:rand.uniform * 25 + 65) |> round end) |> to_string
```

Comment:
hey so it seems `:rand.uniform` isn't available in otp 17, should we have a module/function to decorate this, or kill support for otp 17